tableauserverclient/models/task_item.py

```diff
@@ -20,20 +26,31 @@ def from_response(cls, xml, ns):
         parsed_response = ET.fromstring(xml)
         all_tasks_xml = parsed_response.findall(
             './/t:task/t:extractRefresh', namespaces=ns)
+        all_tasks_xml.extend(parsed_response.findall(
```

Comment:
I think it would be better to pass in the task type here and parse just the one type.

Reply:
Changed it as suggested.